apps/examples/webserver_test/readme.md

```diff
@@ -0,0 +1,36 @@
+# Webserver Test Application
+
+### Features
+
+- HTTPS (based on mbedTLS h/w accelerate)
+- GET/PUT/POST/DELETE method
+- Supports Keep-alive
+- Chunked Transfer
+- Trailer header
+
+### Build Configuration
+
+> rtl8721csm / security_hal_test_tz
```

Comment:
Is this example working only with rtl8721csm?

Reply:
Changes done as suggested